### PR TITLE
fix: proper redirection for forked project links

### LINKED ISSUE #4228: 🐞 Bug: the link to the forked project always redirects to a 404 error page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

clicking on forked projects link always redirects to 404 error page

### Steps To Reproduce

1. go to https://circuitverse.org
2. go to dashboard
3. view any forked project
4. click on the Forked from link
5. see error

### Expected Behavior

The link should redirect to the correct project from which it was forked.

### Screenshots

https://github.com/CircuitVerse/CircuitVerse/assets/91966855/05ac952f-a258-404f-a484-9932d5beca19

### Device Information [optional]

```markdown
- OS: MacOs
- Browser: Safari
- version: 17.1
```

### Additional context

_No response_

### Are you working on this issue?

Yes

### PR DESCRIPTION
**Pull request by niladrix719**
Fixes #4228

#### Describe the changes you have made in this PR -

changed request to get from post

### Screenshots of the changes (If any) -

https://github.com/CircuitVerse/CircuitVerse/assets/91966855/0e59981b-e3c0-40b2-b927-cfd941e0d59b

working as expected now

### CODE REVIEW

**Comment by niladrix719** [comment]
cc @tanmoysrt changing it to get solves it

**Comment by tanmoysrt** [comment]
> cc @tanmoysrt changing it to get solves it

fine ! the Rails CI has some issue, so reviewing it after fixing that.

**Review by tanmoysrt — APPROVED** [review]
Working as expected 
cc @tachyons @vedant-jain03

File: app/views/projects/show.html.erb [hunk 1]
```diff
@@ -67,7 +67,7 @@
           <% if !@project.forked_project.nil? %>
             <p>
               <strong><%= t("projects.show.project_forked_from") %> </strong>
-              <%= link_to "#{@project.forked_project.author.name}/#{@project.forked_project.name}", user_project_path(@project.forked_project.author, @project.forked_project), class: "anchor-text", method: :post %>
+              <%= link_to "#{@project.forked_project.author.name}/#{@project.forked_project.name}", user_project_path(@project.forked_project.author, @project.forked_project), class: "anchor-text", method: :get %>
```
Comment (tachyons) [inline app/views/projects/show.html.erb:70]:
```suggestion
              <%= link_to "#{@project.forked_project.author.name}/#{@project.forked_project.name}", user_project_path(@project.forked_project.author, @project.forked_project), class: "anchor-text" %>
```

**Review by vedant-jain03 — APPROVED** [review]

**Pull request merged**